src/microbe_stage/MicrobeAI.cs

```diff
@@ -397,6 +139,29 @@ private FloatingChunk GetNearestChunkItem(List<FloatingChunk> allChunks)
             }
         }
 
+        // Don't bother with chunks when there's a lot of microbes to compete with
+        if (chosenChunk != null)
```

Comment:
Shouldnt a species with high opportunism still do this.

Reply:
I thought about that, but the issue is that the deathballs that this check is meant to avoid tend to be monospecific, so if any microbe was willing to do it, all of them will, resulting in 10-20 critters getting clumped together. I could see a high DC opportunism check serving as a way to make it only happen sometimes...